AmplifyTools/AmplifyXcode/Sources/AmplifyXcodeCore/Commands/CommandImportModels.swift

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import PathKit
+
+enum CommandImportModelsTasks {
+    static func projectHasGeneratedModels(environment: AmplifyCommandEnvironment,
+                                          args: CommandImportModels.TaskArgs) -> AmplifyCommandTaskResult {
+        let modelsPath = environment.path(for: args.generatedModelsPath)
+        if environment.directoryExists(atPath: modelsPath) {
```

Comment:
nit: It's a bit of a stylistic thing, but I prefer to use `guard` conditions to indicate early exit conditions or non-happy-path flows. In this case, the happy path is that the directory exists, so the failure condition could be modeled as a `guard` with a `return .failure(...)`. As mentioned, it's a stylistic preference, but see what you think.

Reply:
I usually rely on `guard` mostly to indicate early exit conditions, but your style recommendation is a good call so I'll apply it.